protocols/imap/src/test/java/org/apache/james/imap/processor/base/MailboxEventAnalyserTest.java

```diff
@@ -338,7 +338,10 @@ public MailboxPath getMailboxPath() {
                 }
             };
         }
-        
+
+        public MessageManager getMailbox(MailboxId mailboxId, MailboxSession session) throws MailboxException {
+            return getMailbox((MailboxPath)null, session);
```

Comment:
Just because getMailbox(MailboxPath, MailboxSession) was returning one too. But I can return null if you prefer.